admin_pages/events/Events_Admin_Page.core.php

```diff
@@ -1240,17 +1275,17 @@ protected function _default_tickets_update(EE_Event $evtobj, $data)
                 }
             }
             $DTM->save();
-            $DTT = $evtobj->_add_relation_to($DTM, 'Datetime');
+            $DTM = $evtobj->_add_relation_to($DTM, 'Datetime');
```

Comment:
`_add_relation_to()` returns the object you passed in, and since objects are always passed by reference in PHP... THERE WAS NO REASON OR LOGIC BEHIND CHANGING THE VAR NAME !!!